pkg/controller.v2/trainjob_controller.go

```diff
@@ -15,3 +15,46 @@ limitations under the License.
 */
 
 package controllerv2
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
```

Comment:
I think, we agree before to use controller-runtime logger everywhere to be consistent: https://github.com/kubeflow/training-operator/issues/2048
Should we make that change @tenzen-y ?

Reply:
Actually, this is the controller-runtime logger. The controller-runtime logger implement this logr interface, so we need to import this library to use the controller-runtime logger. You can see the actual controller-runtime logger in the Reconcile function.

Reply:
Yeah, that's right, what is the value to configure controller runtime logger with ` klog.KObj(&trainJob)` ?